### DISCUSSION TITLE
Remote URL in NovaNotification opening in new window/tab

### DISCUSSION THREAD

**Discussion by Jaspur** [laravel/nova-issues#5544] [Ideas & Feature Requests] [answered]
Hi,

When I add a downlink now to a NovaNotification, it always opens in the same window. 

```
use Laravel\Nova\URL;

   User::find($fields->user)->notify(
            NovaNotification::make()
                ->message("Invoice is generated")
                ->action('Download', URL::remote($url))
        );
```

Is there a way to open this URL with a `_target="blank"` for example? So after downloading the file, you don't have to navigate back with the browser button.

**Comment by crynobone** [comment, marked as answer]
https://nova.laravel.com/docs/4.0/customization/notifications.html#opening-remote-action-urls-new-tabs

**Comment by Jaspur** [comment]
@crynobone great! but when will it be released? still cannot use it

**Reply by crynobone** [reply]
Available on latest release. Outdated assets?

**Reply by Jaspur** [reply]
I'm already on latest release [v4.26.3](https://nova.laravel.com/releases/4.26.3), but it's not working.

**Reply by crynobone** [reply]
Submit a bug report with reproducing repository.

**Reply by Jaspur** [reply]
@crynobone in which version is it released? because the release page still tells me that [v4.26.3](https://nova.laravel.com/releases/4.26.3) is te latest version - released in June and now its July.
